packages/dgrid/src/Common.ts

```diff
@@ -24,6 +25,19 @@ export class Common extends HTMLWidget {
     pagination: publish<this, boolean>;
     @publish(false, "boolean", "Enable sorting by column")
     sortable: publish<this, boolean>;
+    @publish("", "string", "Default sort by (Use '-' for descending sort)")
+    sortBy: publish<this, string>;
+
+    protected formatSortBy(): [{ property: string, descending: boolean }] | undefined {
+        let sortBy = this.sortBy();
+        if (!sortBy) return undefined;
+        const descending = sortBy[0] === "-";
```

Comment:
It would be easier to work with if this was a boolean publish parameter `.descending(true)`

Reply:
Changes made as suggested